blocks/columns/columns.js

```diff
@@ -1,6 +1,10 @@
 export default function decorate(block) {
+  const numberOfChildren = block.childElementCount || 0;
   const cols = [...block.firstElementChild.children];
   block.classList.add(`columns-${cols.length}-cols`);
+  if (numberOfChildren > 1) {
+    block.classList.add('add-bottom-space');
+  }
```

Comment:
@nimithshetty17 Is there a reason why this class (`add-bottom-space`) is being added using JavaScript? Ideally, you can apply this fix in just CSS.

```
.columns.content-band.icon-list > div {
  margin-bottom: var(--spacer-layout-04);
}
```